ribasim_qgis/widgets/dataset_widget.py

```diff
@@ -268,8 +268,10 @@ def _write_new_model(self) -> None:
                 [
                     f"starttime = {datetime(2020, 1, 1)}\n",
                     f"endtime = {datetime(2021, 1, 1)}\n",
+                    f'crs = "{self.ribasim_widget.crs.authid()}"\n',
                     'input_dir = "."\n',
                     'results_dir = "results"\n',
+                    'ribasim_version = "2024.9.0"\n',
```

Comment:
This seems rather hardcoded. Can we get this version from a different place? Some python file that we generate, or a pyproject file?

Reply:
Cries in python 🐍 💧